language.settings.xml into .gitignore

db4ple opened the pull request.

db4ple committed:
language.settings.xml into .gitignore

df8oe commented:
I have tried this and ran into big trouble when I wanted to import the project in eclipse oxygen. Interpreting of the three .prefs - files failed and I coult not import project. Only if language.settings.xml was present (and no need of any other file) import was successful...

db4ple committed:
Add experimental RTTY TX Support
- Refactored SoftDDS to support FSK modem implementation
- Removed volatile from AudioDriverState and added it to certain members only.
- Added RTTY TX code with support for MSK (not GMSK) to limit bandwith of signal
- Right now a fixed text is transmitted in an endless loop
- Refactored AudioDriver_TxProcessor to simplify use of SSB modulation for digital modes

df8oe commented:
I merge this - but I revert language.setting.xml into gitignore.
